withings_api/common.py

```diff
@@ -342,6 +342,74 @@ class MeasureGetActivityResponse(NamedTuple):
     offset: int
 
 
+class HeartModel(IntEnum):
+    """Heart model."""
+
+    BPM_CORE = 44
+    MOVE_ECG = 91
+
+
+def new_heart_model(value: Optional[int]) -> HeartModel:
+    """Create enum based on primitive."""
+    return cast(HeartModel, enum_or_raise(value, HeartModel))
+
+
+class HeartWearPosition(IntEnum):
+    """Wear position of heart model."""
+
+    RIGHT_WRIST = 0
+    LEFT_WRIST = 1
+    RIGHT_ARM = 2
+    LEFT_ARM = 3
+    RIGHT_FOOT = 4
+    LEFT_FOOT = 5
+
+
+def new_heart_wear_position(value: Optional[int]) -> HeartWearPosition:
+    """Create enum based on primitive."""
+    return cast(HeartWearPosition, enum_or_raise(value, HeartWearPosition))
+
+
+class HeartGetResponse(NamedTuple):
+    """HeartGetResponse."""
+
+    signal: Tuple[int, ...]
+    sampling_frequency: int
+    wearposition: HeartWearPosition
+
+
+class HeartListECG(NamedTuple):
+    """HeartListECG."""
+
+    signalid: Optional[int]
+    afib: Optional[int]
```

Comment:
Why is signalid and afig Optional? `new_heart_list_ecg()` below expects them to have an int type.
 
This looks like a good place for a new IntEnum named AfibClassification. The docs specify the values:

Value | Description
-- | --
0 | Negative
1 | Positive
2 | Inconclusive